infra/core/security/registry-access.bicep

```diff
@@ -0,0 +1,19 @@
+metadata description = 'Assigns ACR Pull permissions to access an Azure Container Registry.'
```

Comment:
Can you not use avm for this and role?

Reply:
For role assignments, I wasn't sure which avm module was best. I found https://github.com/Azure/bicep-registry-modules/tree/main/avm/ptn/authorization/role-assignment but dont see examples of templates using it. Do you know the closest module?

Reply:
For registry access, given thats part of the container apps pattern, I'm inclined to leave that be.

Reply:
Okay